pkg/server/grpc/tracee.go

```diff
@@ -126,6 +126,7 @@ func convertDefinitionToProto(d events.Definition) *pb.EventDefinition {
 		Version:     v,
 		Description: d.GetDescription(),
```

Comment:
Yeah, I thought about it, but didn't want to remove because it might be misleading, as Threat.Description is important for signatures events definitions and events itself, and Description is important for events that are not signatures, so as signatures are not more than 100 I thought duplicating would be okay

Or you prefer Threat.Description is empty when using the event definition? Be aware that when the event itself is triggered, the Threat on it will have the Description set, so it would be different behavior for the same information, depending it is an event or an event definition